test/erlfmt_SUITE_data/simple_comments.erl

```diff
@@ -160,7 +160,7 @@ call() ->
     end,
     receive
         ok -> ok
-    %% comment 5
+        %% comment 5
```

Comment:
Hmm... This is a bit unfortunate, though I understand why it's happening. I think the case of adding a comment to after like it was here before would be quite common

Reply:
Agreed